code/modules/fishing/fishing_lootpools.dm

```diff
@@ -13,12 +13,20 @@
 	var/minimum_rod_tier = 0
 	/// what tier of rod is the highest to have access to the lootable?
 	var/maximum_rod_tier = INFINITY
+	/// what kind of item is needed as a lure
+	var/obj/item/required_lure = null
 
 /// This proc checks for all the conditionals that could apply to a fishing spot. Modify that for special conditions.
 /datum/fishing_lootpool/proc/check_conditionals(var/mob/user, var/obj/item/fishing_rod/fishing_rod)
 	. = TRUE
 	if(fishing_rod.tier < src.minimum_rod_tier || fishing_rod.tier > src.maximum_rod_tier)
 		return FALSE
+	if (required_lure != null)
+		var/obj/item/lure = null
+		if (length(fishing_rod.storage.stored_items))
+			lure = fishing_rod.storage.stored_items[1]
```

Comment:
Feels like this should be refactored out into a `get_lure()` proc on fishing rods. Improves readability and also opens the possibility for e.g.: 'perma-baited' rods

Reply:
Oh, can do